enterprise/models.py

```diff
@@ -764,3 +775,82 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+@python_2_unicode_compatible
+class EnterpriseIntegratedChannel(TimeStampedModel):
+    """
+    Store information related to integrating with external enterprise systems.
+
+    EnterpriseIntegratedChannel is an external system that the Enterprise Customer uses to manage enterprise
+    related data. Each of these requires an edx plugin in order to send data to these systems, this model stores
+    information related to what and where to send data from the enterprise app.
+    """
+
+    name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Third Party name."))
+    data_type = models.CharField(max_length=100, blank=False, null=False, help_text=_("Data Type"))
+    plugin_endpoint = models.CharField(
+        max_length=255,
+        blank=False,
+        null=False,
+        help_text=_("Internal Plugin endpoint.")
+    )
+
+    class Meta:
+        app_label = 'enterprise'
+        verbose_name = _("Enterprise Integrated Channel")
+        verbose_name_plural = _("Enterprise Integrated Channels")
+        unique_together = (("name", "data_type"),)
+
+    def __str__(self):
+        """
+        Return human-readable string representation.
+        """
+        return "<EnterpriseIntegratedChannel {name} for {data_type} data with id {id}>".format(
+            name=self.name,
+            data_type=self.data_type,
+            id=self.id
+        )
+
+    def __repr__(self):
+        """
+        Return uniquely identifying string representation.
+        """
+        return self.__str__()
+
+
+@python_2_unicode_compatible
+class EnterpriseCustomerPluginConfiguration(TimeStampedModel):
+    """
+    Store information related to integrating with external systems for a particular enterprise customer.
+
+    EnterpriseCustomerPluginConfiguration stores the relationship between EnterpriseCustomer
+    and EnterpriseIntegratedChannel, representing which IntegratedChannels are configured for an Enterprise and
+    what data to send where for a given Enterprise.
+    """
```

Comment:
Do you need a separate model for this? It looks like this is just storing the relationship. You could accomplish the same thing by adding a foreign key to either `EnterpriseCustomer` or `EnterpriseIntegratedChannel`.